src/features/create-dao/Steps/TreasuryForm.tsx

```diff
@@ -1,24 +1,56 @@
 import { FC, useContext } from 'react';
 
+import * as Yup from 'yup';
 import { Form, Formik } from 'formik';
 
 import { Wizard } from '@zero-tech/zui/components';
 import { CreateDAOFormContext } from '../';
 
 import styles from './TreasuryForm.module.scss';
+import { WrappedInput } from '../../ui/WrappedInput/WrappedInput';
+import classNames from 'classnames/bind';
 
-interface TreasuryFormProps {
+const validationSchema = Yup.object().shape({
+	gnosisSafe: Yup.string().required('Gnosis Safe is required'),
```

Comment:
Let's update the validation message here to 'The gnosis safe field is required.'.

Reply:
Updated